src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/index.js

```diff
@@ -0,0 +1,121 @@
+import {Box, Flex} from 'grid-emotion';
+import React from 'react';
+import {Link} from 'react-router';
+
+import AsyncView from 'app/views/asyncView';
+import Button from 'app/components/button';
+import CircleIndicator from 'app/components/circleIndicator';
+import EmptyMessage from 'app/views/settings/components/emptyMessage';
+import SentryAppAvatar from 'app/components/avatar/sentryAppAvatar';
+import PropTypes from 'prop-types';
+import {Panel, PanelItem, PanelBody, PanelHeader} from 'app/components/panels';
+import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
+import {t} from 'app/locale';
+import styled from 'react-emotion';
+import space from 'app/styles/space';
+import {withTheme} from 'emotion-theming';
+
+class SentryApplicationRow extends React.PureComponent {
+  static propTypes = {
+    app: PropTypes.object.isRequired,
+    orgId: PropTypes.string.isRequired,
+  };
+
+  render() {
+    let {app, orgId} = this.props;
+    let btnClassName = 'btn btn-default';
+
+    return (
+      <PanelItem justify="space-between" px={2} py={2}>
```

Comment:
Let's use styled components instead of these `px` `py` etc props (same thing with the `Box` components below